direct/src/dist/commands.py

```diff
@@ -209,8 +173,7 @@ def get_data(path):
 del os
 """
 
-SITE_PY = SITE_PY3 if sys.version_info >= (3,) else SITE_PY2
-
+SITE_PY = SITE_PY3 if sys.version_info >= (3,) 
```

Comment:
This is not actually valid Python syntax.  You don't need the `if` here.  In fact, you don't need the alias.  I suggest just removing this line and changing the original `SITE_PY3` to `SITE_PY`.

Reply:
Sure I will sure do it.

Reply:
Sure ! I will make those changes.